src/Application.vala

```diff
@@ -108,9 +108,11 @@ public class AppCenter.App : Granite.Application {
         }
 
         if (main_window == null) {
-            client.update_cache.begin (true);
-
             main_window = new MainWindow (this);
+            // Start updating the cache 5 seconds after starting the main window
+            // If we do it instantly, we slow down the loading of the banner on the
+            // homepage considerably
+            schedule_cache_update (false, 5);
```

Comment:
And when I say interfering, I mean slowing each other down. Nothing crashes, just the less important check for update actions tend to get put at the front of the queue, and the banner takes a while to change as a result.